doc/workflows_and_tools/enrollment_wizard/index.rst

```diff
@@ -44,3 +44,19 @@ wizard in your html templates defined in these files:
    your needs the best by defining a variable PI_CUSTOMIZATION in the file
    pi.cfg. This way you can put all modifications in one place apart from the
    original code.
+
+Example
+~~~~~~~
+
+Your privacyIDEA system is running in the URL sub path ``/pi``.
+You want the files to be addressed via the URL path ``/mydesign``.
```

Comment:
Hmm, I don't really understand this sentence. Don't we want the files to be addressed via the URL path ``/pi/mydesign``?

Reply:
Yes, actually the files are available on the webserver via "/pi/mydesign".

Reply:
Ah okay. Then, I find the sentence you suggested better understandable, because ``/mydesign`` reads like an absolute path to me.

Reply:
updated.